openmdao/surrogate_models/multifi_cokriging.py

```diff
@@ -998,12 +998,13 @@ def train_multifi(self, X, Y):
 
         Parameters
         ----------
-        X : array_like
-            An array with shape (n_samples_X, n_features) with the input at which observations
-            were made.
-        Y : array_like
-            An array with shape (n_samples_X, n_features) with the observations of the scalar
-            output to be predicted.
+        X : list of double array_like elements
+            A list of arrays with the input at which observations were made, from highest
+            fidelity to lowest fidelity. Designs must be nested
+            with X[i] = np.vstack([..., X[i+1])
```

Comment:
Wait, is x really ordered from high to low fidelity, but y is from low to high?

Reply:
Thanks for catching that!

Reply:
I think the lowest fidelity is the first one. See the docstring for the `fit` method and notice that in the code they normalize by the lowest fidelity, which is in index 0.

Reply:
The `train_multifi()` method of the `MultiFiCoKrigingSurrogate` class reverses the order of the lists of arrays before feeding them to the `fit()` method of the `MultiFiCoKriging` class. That means that the *highest* fidelity is the first one, not the lowest.